docs/core/tutorials/using-on-macos.md

```diff
@@ -129,46 +96,45 @@ namespace Library
 }
 ```
 
-This makes use of a number of modern C# features, such as 
-static usings, expression-bodied members, and interpolated strings,
-that you can learn
-about in the [Learn C#](../../csharp/index.md) section.
+The `Thing` class contains one public method, `Get`, which returns the sum of two numbers but does so by converting the sum into a string and then deserializing it into an integer. This makes use of a number of modern C# features, such as static `using` directives, expression-bodied members, and interpolated strings, that you can learn about in the [Learn C#](../../csharp/index.md) section.
```

Comment:
I'd remove the "that you can learn about..." parenthetical and replace it with links to the C# topics themselves. They are (I think the links are all correct):
- [using static](../../csharp/language-reference/keywords/using-static.md).
   Note there that it's not status `using` directive, but `using static` directive.
- [expression-bodied members](../../csharp/csharp-7.md#more-expression-bodied-members]
   Unfortunately, here we don't have a separate topic (I just opened an issue for it), but there's at least some code.
- [interpolated strings](../../csharp/language-reference/keywords/interpolated-strings.md)